README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $(GOPATH)/github.com/opencontainers/runc
```

Comment:
I noticed that the Godeps directory is existed in the repository, and the `go get` is slow and unreliable, as it does not provide repeatable builds.